plutus-core/untyped-plutus-core/test/Evaluation/ApplyBuiltinName.hs

```diff
@@ -48,7 +48,7 @@ type AppErr =
 newtype AppM a = AppM
     { unAppM :: Either AppErr a
     } deriving newtype (Functor, Applicative, Monad, MonadError AppErr)
-      deriving stock (MonadEmitter) via (NoEmitterT AppM)
+    deriving (MonadEmitter) via (NoEmitterT AppM)
```

Comment:
stylish-haskell was unable to parse this.
This suggests that `deriving stock X via Y` is wrong, even though GHC doesn't complain.
I've built both version and the two resulting `ApplyBuiltinName.o` files are bit-identical